src/cogs/admin_commands.py

```diff
@@ -89,7 +91,42 @@ async def delete_finished_surveys_channels(self, ctx):
 
             logging.info(f"Deleting channel ({channel_id}).")
             await channel.delete()
-            db.set_user_survey_progress_status_to_channel_deleted(channel_id)
+            db.set_user_survey_progress_status(
+                channel_id, survey_status.FINISHED_CHANNEL_DELETED
+            )
+            deleted_channels.append(channel_id)
+
+        await ctx.channel.send(f"Deleted {len(deleted_channels)} channels.")
+
+    @commands.command(name="delete-unanswered-surveys-channels")
+    @commands.has_role(ADMIN_ROLE_ID)
+    async def delete_unanswered_surveys_channels(self, ctx):
+        """
+        Only member with admin role can run this command.
+        Command deletes channel of surveys which have been unanswered longer than DELETE_UNANSWERED_SURVEYS_OLDER_THAN
+        """
+        deleted_channels = []
+        logging.info(
+            f"Executing delete_unanswered_surveys_channels command. OLDER_THAN is set to {DELETE_UNANSWERED_SURVEYS_OLDER_THAN}"
+        )
+        unanswered_channel_ids = (
+            db.get_in_progress_survey_channel_ids_created_older_than(
+                DELETE_UNANSWERED_SURVEYS_OLDER_THAN
+            )
+        )
+        for channel_id in unanswered_channel_ids:
```

Comment:
Perhaps the channel deletions could be extracted into a common function reused in `delete_finished_surveys_channels` e.g. `delete_channels_and_set_survey_status` or simply `delete_survey_channels`.